internal/plumbing/ticks_test.go

```diff
@@ -203,11 +203,13 @@ func TestTicksSinceStartConsumeZero(t *testing.T) {
 	// depending on where the contributor clones this project from, the remote
 	// reported in the error could either be from gopkg.in or github.com
 	if !strings.Contains(output, "github.com") && !strings.Contains(output, "gopkg.in") {
-		assert.Failf(t, "output should contain either 'github.com' or 'gopkg.in':\n%s", output)
+		assert.Failf(t, "output should contain either 'github.com' or 'gopkg.in'", "got: '%s'", output)
 	}
 	assert.Equal(t, res[DependencyTick].(int), 0)
 	assert.Equal(t, tss.previousTick, 0)
-	assert.Equal(t, tss.tick0.Year(), 1969)
+	if (tss.tick0.Year() != 1969) && (tss.tick0.Year() != 1970) {
```

Comment:
The root of the problems seems to be that `commit.Committer.When` is initialized to `time.Unix(0, 0)` and that time must be GMT. It is subsequently converted to the local time and we get 1969. Should be fine here.